src/Lykke.Service.HftInternalService/Modules/ServiceModule.cs

```diff
@@ -38,31 +35,22 @@ protected override void Load(ContainerBuilder builder)
             RegisterApiKeyService(builder);
 
             BindMongoDb(builder);
-            BindRedis(builder);
+            BindRabbitMq(builder);
         }
 
 
-        private void BindRedis(ContainerBuilder builder)
+        private void BindRabbitMq(ContainerBuilder builder)
         {
-            var apiKeysRedisCache = new RedisCache(new RedisCacheOptions
-            {
-                Configuration = _settings.CurrentValue.HighFrequencyTradingService.CacheSettings.RedisConfiguration,
-                InstanceName = _settings.CurrentValue.HighFrequencyTradingService.CacheSettings.ApiKeyCacheInstance
-            });
-            builder.RegisterInstance(apiKeysRedisCache)
-                .As<IDistributedCache>()
-                .Keyed<IDistributedCache>("apiKeys")
-                .SingleInstance();
+            builder.RegisterType<ApiKeyPublisher>()
+                .As<IApiKeyPublisher>()
+                .As<IStartable>()
+                .SingleInstance()
```

Comment:
Should also register AsStopable? Is there anyone who can call Stop?

Reply:
Sure.